controllers/user.js

```diff
@@ -19,10 +20,10 @@ module.exports.post = async function post (req, res) {
 
 module.exports.put = async function put (req, res) {
   await UserModel.findOneAndUpdate({ _id: req.params.id }, req.body)
-  res.send()
+  res.send('User modified: ' + res.locals.name)
 }
 
 module.exports.delete = async function del (req, res) {
   await UserModel.findOneAndUpdate({ _id: req.params.id }, {isActive: false})
-  res.send()
+  res.send('User deleted: ' + res.locals.name)
```

Comment:
i will bet to use res.json() all the time instead of send()

Reply:
perfect, thanks!